wikibase/1.31/.travis/build-deploy.sh

```diff
@@ -2,8 +2,8 @@
 #Oneline for full directory name see: https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 set -e
-docker build "$DIR/../base" -t wikibase/wikibase:1.31 -t wikibase/wikibase:1.31-base -t wikibase/wikibase:lts -t wikibase/wikibase:lts-base
-docker build "$DIR/../bundle" -t wikibase/wikibase:1.31-bundle -t wikibase/wikibase:lts-bundle
+docker build "$DIR/../base" -t wikibase/wikibase:1.31 -t wikibase/wikibase:1.31-base
+docker build "$DIR/../bundle" -t wikibase/wikibase:1.31-bundle
 
 if [ "$SHOULD_DOCKER_PUSH" = true ]; then
```

Comment:
I’m not sure if I understand this correctly, but shouldn’t the three “lts” lines in this block move to the 1.35 file?

Reply:
They seem to? (Not sure if I'm looking at a different version of the file / pr?)

Reply:
The block _below_ my comment, within the SHOULD_DOCKER_PUSH, still has lts lines in the 1.31 file, unless I’m misreading the git diff.

Reply:
+1 yes these should be removed!
(github UI made that hard to see)

![image](https://user-images.githubusercontent.com/3308769/98375038-74d0d580-2039-11eb-91fc-ff7a23005738.png)